src/code/z_collision_check.c

```diff
@@ -3096,9 +3096,9 @@ s32 CollisionCheck_LineOC_JntSph(GlobalContext* globalCtx, CollisionCheckContext
         if (!(element->info.ocElemFlags & OCELEM_ON)) {
             continue;
         }
-        D_8015E610.a = *a;
-        D_8015E610.b = *b;
-        if (Math3D_LineVsSph(&element->dim.worldSphere, &D_8015E610) == 1) {
+        lineSeg.a = *a;
+        lineSeg.b = *b;
+        if (Math3D_LineVsSph(&element->dim.worldSphere, &lineSeg) == true) {
```

Comment:
At first I thought `lineSeg` should be `line` ("infinite" line, not a line segment), but looking at `Math3D_LineVsSph` if I got it right it should actually be `Math3D_LineSegVsSph`?
the comment on `CollisionCheck_LineOC_JntSph` also says "line" so this is confusing

Reply:
seems more an issue for Math3D, since the system defines an InfiniteLine type

Reply:
I'm going to hold off on resolving this issue in this PR I think, there's several issues with (infinite) line vs line segment and sys_math3d could probably do with some other improvements too, so this would be best addressed in a sys_math3d focused cleanup (which I may do at some point but don't wait for me if anyone else wants to)